packages/webapp/src/delegates/api/eden-contract.ts

```diff
@@ -18,14 +15,39 @@ const getMemberBudgetBalance = () => {
     return {}; // TODO
 };
 
-const getMemberElectionParticipationStatus = () => {
-    return {}; // TODO
-};
+export const getMemberRecordFromName = (
```

Comment:
Since i'm using this in non-UI code, I'm going to leave it as is. I could obviously add a hook that uses the helper function if we think it'll be useful other places. Right now, it's just this same helper function called in 2 parts of the the UI code. Lmk if you feel strongly enough about creating the hook now, and I'll add it. Otherwise, I'll leave it until we find we're needing it more.

Reply:
Nah, I don't feel strongly about it. That's totally fine. 👍